src/locales/cy/default.yml

```diff
@@ -63,14 +63,14 @@ error:
     buttonText: "Ewch i hafan GOV.UK"
     buttonLink: "https://www.gov.uk/"
 links:
-  changeAddress: '<a href="/address/edit" id="change-address" data-id="changeAddress" class="govuk-link">Newid<span class="govuk-visually-hidden"> eich cyfeiriad presennol</span></a>'
-  changeYearFrom: '<a href="/address/edit" id="change-address" data-id="changeAddress" class="govuk-link">Newid<span class="govuk-visually-hidden"> y flwyddyn y dechreuoch chi fyw yn eich cyfeiriad presennol</span></a>'
+  changeAddress: '<a href="/address?edit=true" id="change-address" data-id="changeAddress" class="govuk-link">Newid<span class="govuk-visually-hidden"> eich cyfeiriad presennol</span></a>'
```

Comment:
You can't change the path. I suppose it's link to a specific route. 
What we want is to add a new url query param: https://www.npmjs.com/package/@govuk-one-login/one-login-analytics#form-change-tracker
This comment applies to all changes related to the path ".../edit"

Reply:
Reverted the swap, query parameter added as an addition rather than a replacement